t/mojo/daemon.t

```diff
@@ -18,13 +18,21 @@ use Mojolicious;
 package TestApp;
 use Mojo::Base 'Mojolicious';
 
+my $message = 'Test';
+
 sub handler {
   my ($self, $tx) = @_;
   $tx->res->code(200);
-  $tx->res->body('Hello TestApp!');
+  $tx->res->body("Hello $message!");
```

Comment:
Why are the quotes changed?

Reply:
Because we need variable interpolation now!

Reply:
Anyway I'm going to remove this code as it is not needed.